### DISCUSSION TITLE
Too few reads used in clonotypes

### DISCUSSION THREAD

**Discussion by NoELJ22** [milaboratory/mixcr#1161] [Q&A]
Hello,

Could you help me understand why there is so low reads used in clonotypes ?
`Reads used in clonotypes, percent of total: 13696 (1.73%)`

We're working on human PBMC cultured in vitro. The sample isn't stimulated by Ag (Day 0).  
For sequencing, the Takara SMART-Seq Human BCR IgG IgM IgA IgD IgE H/K/L Profiling Kit was used. We only used the primer for IgH

The alignment looks good and the overlapping too
`Successfully aligned reads: 702545 (88.53%)`
`Overlapped: 661538 (83.36%)`

Code used : 
```
mixcr analyze takara-human-bcr-full-length \
   --remove-step exportClones \
   123-J0-unst_S8_L001_R1_001.fastq.gz 123-J0-unst_S8_L001_R2_001.fastq.gz 123_J0_unstim

mixcr exportReports 123_J0_unstim.clns 123_J0_unstim.report.txt 
```

I attach the report and fastQC files via filesender:
[123_J0_unstim.report.txt](https://github.com/milaboratory/mixcr/files/11786033/123_J0_unstim.report.txt)
https://filesender.renater.fr/?s=download&token=60478198-bf78-47cd-83db-caf1f722b35a

Thank you for your help

**Comment by PoslavskySV** [comment]
The issue was resolved via support@milaboratories.com (something went wrong in the wet lab causing missing nucleotides of UMI; workaround was to analyze without UMIs)
